var/spack/repos/builtin/packages/ports-of-call/package.py

```diff
@@ -10,13 +10,13 @@ class PortsOfCall(CMakePackage, CudaPackage):
     """Ports of Call: Performance Portability Utilities"""
 
     homepage    = "https://github.com/lanl/ports-of-call"
-    url         = "https://github.com/lanl/ports-of-call/archive/refs/tags/v1.0.0.tar.gz"
+    url         = "https://github.com/lanl/ports-of-call/archive/refs/tags/v1.1.0.tar.gz"
     git         = "https://github.com/lanl/ports-of-call.git"
 
     maintainers = ['rbberger']
 
     version("main", branch="main")
-    version('1.0.0', sha256='4f451a41baeb2f95d30760cf39279785dcd6e550b3ce5eac05ef0a258af67e03')
+    version('1.1.0', sha256='c47f7e24c82176b69229a2bcb23a6adcf274dc90ec77a452a36ccae0b12e6e39')
```

Comment:
Please leave old versions as part of the recipe for the sake of reproducibility.

Reply:
@sethrj The 1.0.0 version had missing CMake files, which means packages that depend on it will fail. Only "main" had the proper files, which I is why I asked the devs for a new `1.1.0` release. I can keep 1.0.0 in if you insist, but it will be mostly unusable.